### PR TITLE
Only o4s missing now

### PR DESCRIPTION
**Pull request by ArkfrostLumas**
Did Rabanastre and o1s-o3s. I've also corrected a little mistake in o8s. 
For some reason there are conflicts and I looked into them, but my changes are alright.
Got a question about a certain case through. If you look at rabanastre line 186:

if (data.maskValue)
          return {
            en: 'Move',
            de: 'Bewegen',
          };
        return {
          en: 'Stop',
          de: 'Stopp',
        };

I'm not sure how to place the curly braces there since those are two different cases. The original was:

if (data.maskValue)
          return  'Move',;
        return 'Stop';
 },

And those two returns seem to be separate depending on something else, or am I mistaken there?

### CODE REVIEW

**Comment by quisquous** [comment]
This:
```
      infoText: function(data) {
        if (data.maskValue)
          return 'Move';
        return 'Stop';
      },

```

...is the same as this:
```
      infoText: function(data) {
        if (data.maskValue) {
          return 'Move';
        }
        return 'Stop';
      },
```

... so you should do something like:
```
      infoText: function(data) {
        if (data.maskValue) {
          return {
            en: 'Move',
          };
        }
        return {
          en: 'Stop',
        };
      },
```

Does that help clarify?

For what it's worth, the tsuku and ridorana (and many deltascape era things) do not have translated timelines either.  They all need a timelineReplace section at the bottom.  I'll try to do an autotranslation pass this weekend, but you're welcome to give a pass too if you want.

**Comment by quisquous** [comment]
Re: rabanastre.

```
diff --git a/ui/raidboss/data/triggers/royal_city_of_rabanastre.js b/ui/raidboss/data/triggers/royal_city_of_rabanastre.js
index e855eb9..d3d28cd 100644
--- a/ui/raidboss/data/triggers/royal_city_of_rabanastre.js
+++ b/ui/raidboss/data/triggers/royal_city_of_rabanastre.js
@@ -117,7 +117,7 @@
       regex: /:Archaeolion starts using The Dragon's Voice/,
       regexDe: /:ArchaeolÃ¶we starts using Stimme Des Drachen/,
       alertText: {
-        en: 'Dragon Voice: Move In'
+        en: 'Dragon Voice: Move In',
         de: 'Stimme Des Drachen: Rein',
       },
       tts: {
@@ -226,10 +226,11 @@
           en: 'Look Away',
           de: 'Wegschauen',
         };
-      return {
-        en: 'Look Towards',
-        de: 'anschauen',
-      };
+	return {
+	  en: 'Look Towards',
+	  de: 'anschauen',
+	};
+      },
     },
   ],
 }]
```

**Comment by ArkfrostLumas** [comment]
Ahhhh, okay. That makes much more sense now.
So another way to compare it is, that:

 infoText: function(data) {
        if (data.maskValue)
          return 'Move';
        return 'Stop';
      },

Is basically:

 infoText: function(data) {
        if (data.maskValue) return 'Move';  //the if only counts for this return  
        return 'Stop';
      },

Then everything makes a lot more sense. Gonna edit all missing curly braces now :)
And thanks for looking up the rabanastre issue. Now that you pointed it out I also noticed it ^^

**Comment by ArkfrostLumas** [comment]
I was a little bit overenthusiastic and did **everything** now xD

For some reason git bash didn't fetch the changes from upstream that were made within the closed german-update-byakko, which is why I redid the whole byakko and eureka-anemos (and also fixed some typos and wording issues)

Also I had to correct a bunch of typos in some files, which is why there are so many conflicts. I've checked all of them and you can overwrite everything I've changed. Everything seems to be fine.

I've also tested all the new scripts within ACT and I'm not getting a single error message, which sounds promising :)

File: ui/raidboss/data/triggers/o2s.js [hunk 1]
```diff
@@ -16,93 +17,202 @@
...
       id: 'O2S Levitation',
-      regex: /:(\y{Name}) gains the effect of (?:Unknown_556|Levitation) from/,
+      regex: /:(\y{Name}) gains the effect of Levitation from/,
       condition: function(data, matches) { return matches[1] == data.me; },
       run: function(data) { data.levitating = true; },
     },
     {
       id: 'O2S Levitation',
-      regex: /:(\y{Name}) loses the effect of (?:Unknown_556|Levitation)/,
+      regex: /:(\y{Name}) loses the effect of Levitation/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o2s.js]: Did you miss a regexDe here too?

File: ui/raidboss/data/triggers/o4s.js [hunk 2]
```diff
@@ -153,113 +198,202 @@
...
-          return 'Be Left On Blue';
-        return 'Color sides';
+        }
+        if (data.blackWound) {
+          return {
+            en: 'Be Right On Purple',
+            de: 'Rechts auf Pink',
+          };
+        }
+        else if (data.whiteWound) {
```
[the remark this answers is not in the document]
Comment (quisquous) [reply ui/raidboss/data/triggers/o4s.js]: Still missing all of these in this file, sorry.

File: ui/raidboss/data/triggers/o7s.js [hunk 3]
```diff
@@ -35,10 +35,12 @@
       regexJa: / 1A:ガーディアン gains the effect of ダダルマー・プログラム/,
       condition: function(data, matches) { return !data.first || data.seenVirus && !data.second },
       run: function(data) {
-        if (data.seenVirus)
+        if (data.seenVirus) {
           data.second = 'dada';
-        else
+        }
+        else {
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o7s.js]: elses on the wrong line in this file too (in multiple spots)

File: ui/raidboss/data/triggers/unending_coil_ultimate.js [hunk 4]
```diff
@@ -527,12 +535,15 @@
       run: function(data, matches) {
         data.dooms = data.dooms || [null, null, null];
         var order = null;
-        if (data.ParseLocaleFloat(matches[2]) < 9)
+        if (data.ParseLocaleFloat(matches[2]) < 9) {
           order = 0;
-        else if (data.ParseLocaleFloat(matches[2]) < 14)
+        }
+        else if (data.ParseLocaleFloat(matches[2]) < 14) {
```
Comment (quisquous) [inline ui/raidboss/data/triggers/unending_coil_ultimate.js]: elses on the wrong line in this file too

File: ui/eureka/eureka.js [hunk 5]
```diff
@@ -276,7 +276,8 @@ var Options = {
   },
 }
 
-var gFlagRegex = Regexes.Parse(/00:00..:(.*)Eureka (?:Anemos|Pagos) \( (\y{Float})  , (\y{Float}) \)(.*$)/);
+//var gFlagRegex = Regexes.Parse(/00:00..:(.*)Eureka (?:Anemos|Pagos) \( (\y{Float})  , (\y{Float}) \)(.*$)/);
```
Comment (quisquous) [inline ui/eureka/eureka.js]: Can you remove this commented line too and just leave your edit?

File: ui/raidboss/data/triggers/o2s.js (hunk 1 above)
Comment (ArkfrostLumas) [reply ui/raidboss/data/triggers/o2s.js]: Nope, it's also Levitation in German, so this doesn't need a RegexDe :>

**Comment by ArkfrostLumas** [comment]
I think now I've caught all what was left

**Comment by quisquous** [comment]
Looks great! Thanks for all the changes.  :)

**Pull request merged**